azure/service.go

```diff
@@ -203,18 +210,20 @@ func GetNewSession(ctx context.Context, d *plugin.QueryData, tokenAudience strin
 	sess := &Session{
 		Authorizer:              authorizer,
 		CloudEnvironment:        settings.Environment.Name,
-		Expires:                 &expiresOn,
+		Expires:                 expiresOn,
 		GraphEndpoint:           settings.Environment.GraphEndpoint,
 		ResourceManagerEndpoint: settings.Environment.ResourceManagerEndpoint,
 		StorageEndpointSuffix:   settings.Environment.StorageEndpointSuffix,
 		SubscriptionID:          subscriptionID,
 		TenantID:                tenantID,
 	}
 
-	if sess.Expires != nil {
+	if expiresOn != nil {
 		d.ConnectionManager.Cache.SetWithTTL(cacheKey, sess, time.Until(*sess.Expires))
+		logger.Info("Session saved in cache with expiry in", "minutes", (time.Until(*sess.Expires)).Minutes())
```

Comment:
Similar to my other questions above, should this line and the one below it be debug level?